src/components/Logo/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './index.css'
+import keyboardIcon from '../../assets/icons/keyboard-icon.svg'
+function Logo(){
+return (
+    <div className="Logo">
+            <img className="keyboardLogo" src={keyboardIcon} alt="logo" />
+            <div className="logoTitle"> fast finger</div>
+            <div className="logoDescription">____________the ultimate typing game___________</div>
```

Comment:
```suggestion
            <LabeledHR>The ultimate typing game</LabeledHR>
```
JS

    function LabeledHR({ children }) {
      return (
        <div className="labeled-hr">
          <div></div>
          <span>{children}</span>
          <div></div>
        </div>
      );
    }

SCSS

    .labeled-hr {
      display: flex;
      width: 80%;
      align-items: center;
    
      div {
        flex: 1;
        height: 1px;
        background-color: #ffffff;
        margin: 0 1em;
      }
    
      span {
        text-transform: uppercase;
        font-weight: bold;
      }
    }

Reply:
awesome I didn't think that this could be achieved with css. How about using bottom- border rather than background-color Any suggestions ?

Reply:
Right, you use `border-bottom` as well. In that case, make sure to keep the `height: 0px` or else it won't exactly be in the center